tools/pythonpkg/tests/fast/arrow/test_tpch.py

```diff
@@ -7,20 +7,31 @@
 except:
     can_run = False
 
+def answer_munge(cell):
+    try:
+        cell = round(float(cell), 2)
+    except ValueError:
+        cell = str(cell)
+    return cell
+
+def result_munge(cell):
+    try:
+        cell = round(float(cell), 2)
+    except (ValueError, TypeError):
+        cell = str(cell)
+    return cell
```

Comment:
These two could probably be collapsed, and there are probably better ways to do this — but the main purpose of them is to take the all-text inputs of the answers or the types from results and convert them (where possible) into comparable types (rounding so that small differences are ignored — which is mentioned in the previous code).

This (semi-blind) rounding isn't quite right for true TPC-H answer verification, but works well enough for these kinds of tests.

I'm more than happy to clean this up.

Reply:
Can't we just use ```result_munge``` for both cases?

Reply:
Yup, totally

Reply:
Done